<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Server IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="server">
       <property name="text">
        <string>192.168.56.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="port">
       <property name="maximum">
        <number>999999</number>
       </property>
       <property name="value">
        <number>12345</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="nickname">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>nick name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConnecter">
       <property name="text">
        <string>Connect Server</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="chatLog"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="message">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>45</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>message</sender>
   <signal>returnPressed()</signal>
   <receiver>btnSend</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>451</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
